SDK build error on a machine having only VS14

On a machine which only has VS14 and VSSDK14, I tried to do a sdk build but the VSIX generation part failed. Basically, it tries to find C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v12.0\VSSDK\Microsoft.VsSDK.targets which does not exist on the machine.


The full error log:

"D:\Builds\25093\interpreter\SDK_Buddy\src\setup\ProductSDK\VsixPackages.dirproj" (Build target) (1) ->
(CheckForPrerequisites target) -> 
  D:\Builds\25093\interpreter\SDK_Buddy\src\setup\ProductSDK\VsixPackages.dirproj(82,9): warning NETMF: Visual Studio 2013 SDK not found VSIX package will not be generated


"D:\Builds\25093\interpreter\SDK_Buddy\src\setup\ProductSDK\VsixPackages.dirproj" (Build target) (1) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Framework\CorDebug\vs14\netmfvs14.csproj" (Build target) (2) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ItemTemplates\CSharp\AssemblyInfo\AssemblyInfoTemplateProject.csproj" (default target) (3) ->
  D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ItemTemplates\CSharp\AssemblyInfo\AssemblyInfoTemplateProject.csproj(66,3): error MSB4019: The imported project "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v12.0\VSSDK\Microsoft.VsSDK.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.


"D:\Builds\25093\interpreter\SDK_Buddy\src\setup\ProductSDK\VsixPackages.dirproj" (Build target) (1) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Framework\CorDebug\vs14\netmfvs14.csproj" (Build target) (2) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ItemTemplates\CSharp\Class\ClassTemplateProject.csproj" (default target) (4) ->
  D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ItemTemplates\CSharp\Class\ClassTemplateProject.csproj(64,3): error MSB4019: The imported project "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v12.0\VSSDK\Microsoft.VsSDK.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.


"D:\Builds\25093\interpreter\SDK_Buddy\src\setup\ProductSDK\VsixPackages.dirproj" (Build target) (1) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Framework\CorDebug\vs14\netmfvs14.csproj" (Build target) (2) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ItemTemplates\VisualBasic\Class\VBClassTemplateProject.csproj" (default target) (5) ->
  D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ItemTemplates\VisualBasic\Class\VBClassTemplateProject.csproj(65,3): error MSB4019: The imported project "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v12.0\VSSDK\Microsoft.VsSDK.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.


"D:\Builds\25093\interpreter\SDK_Buddy\src\setup\ProductSDK\VsixPackages.dirproj" (Build target) (1) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Framework\CorDebug\vs14\netmfvs14.csproj" (Build target) (2) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ProjectTemplates\CSharp\ExeTemplate\ExeTemplateProject.csproj" (default target) (6) ->
  D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ProjectTemplates\CSharp\ExeTemplate\ExeTemplateProject.csproj(73,3): error MSB4019: The imported project "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v12.0\VSSDK\Microsoft.VsSDK.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.


"D:\Builds\25093\interpreter\SDK_Buddy\src\setup\ProductSDK\VsixPackages.dirproj" (Build target) (1) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Framework\CorDebug\vs14\netmfvs14.csproj" (Build target) (2) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ProjectTemplates\CSharp\LibTemplate\LibTemplateProject.csproj" (default target) (7) ->
  D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ProjectTemplates\CSharp\LibTemplate\LibTemplateProject.csproj(72,3): error MSB4019: The imported project "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v12.0\VSSDK\Microsoft.VsSDK.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.


"D:\Builds\25093\interpreter\SDK_Buddy\src\setup\ProductSDK\VsixPackages.dirproj" (Build target) (1) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Framework\CorDebug\vs14\netmfvs14.csproj" (Build target) (2) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ProjectTemplates\CSharp\WinTemplate\WinTemplateProject.csproj" (default target) (8) ->
  D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ProjectTemplates\CSharp\WinTemplate\WinTemplateProject.csproj(74,3): error MSB4019: The imported project "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v12.0\VSSDK\Microsoft.VsSDK.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.


"D:\Builds\25093\interpreter\SDK_Buddy\src\setup\ProductSDK\VsixPackages.dirproj" (Build target) (1) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Framework\CorDebug\vs14\netmfvs14.csproj" (Build target) (2) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ProjectTemplates\VB\ExeTemplate\VBExeTemplateProject.csproj" (default target) (9) ->
  D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ProjectTemplates\VB\ExeTemplate\VBExeTemplateProject.csproj(75,3): error MSB4019: The imported project "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v12.0\VSSDK\Microsoft.VsSDK.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.


"D:\Builds\25093\interpreter\SDK_Buddy\src\setup\ProductSDK\VsixPackages.dirproj" (Build target) (1) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Framework\CorDebug\vs14\netmfvs14.csproj" (Build target) (2) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ProjectTemplates\VB\LibTemplate\VBLibTemplateProject.csproj" (default target) (10) ->
  D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ProjectTemplates\VB\LibTemplate\VBLibTemplateProject.csproj(68,3): error MSB4019: The imported project "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v12.0\VSSDK\Microsoft.VsSDK.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.


"D:\Builds\25093\interpreter\SDK_Buddy\src\setup\ProductSDK\VsixPackages.dirproj" (Build target) (1) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Framework\CorDebug\vs14\netmfvs14.csproj" (Build target) (2) ->
"D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ProjectTemplates\VB\WinTemplate\VBWinTemplateProject.csproj" (default target) (11) ->
  D:\Builds\25093\interpreter\SDK_Buddy\src\Product\AllSDK\ProjectTemplates\VB\WinTemplate\VBWinTemplateProject.csproj(74,3): error MSB4019: The imported project "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v12.0\VSSDK\Microsoft.VsSDK.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.

--- a/Product/AllSDK/ItemTemplates/CSharp/AssemblyInfo/AssemblyInfoTemplateProject.csproj
+++ b/Product/AllSDK/ItemTemplates/CSharp/AssemblyInfo/AssemblyInfoTemplateProject.csproj
@@ -25,7 +25,7 @@
     <Configuration Condition="'$(FLAVOR_WIN)'!=''">$(FLAVOR_WIN)</Configuration>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <NetMFBuildVsSdkVersion>12</NetMFBuildVsSdkVersion>
+    <NetMFBuildVsSdkVersion>14</NetMFBuildVsSdkVersion>
     <GeneratePkgDefFile>false</GeneratePkgDefFile>
     <IncludeAssemblyInVSIXContainer>false</IncludeAssemblyInVSIXContainer>
     <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>

--- a/Product/AllSDK/ItemTemplates/CSharp/Class/ClassTemplateProject.csproj
+++ b/Product/AllSDK/ItemTemplates/CSharp/Class/ClassTemplateProject.csproj
@@ -25,7 +25,7 @@
     <Configuration Condition="'$(FLAVOR_WIN)'!=''">$(FLAVOR_WIN)</Configuration>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <NetMFBuildVsSdkVersion>12</NetMFBuildVsSdkVersion>
+    <NetMFBuildVsSdkVersion>14</NetMFBuildVsSdkVersion>
     <GeneratePkgDefFile>false</GeneratePkgDefFile>
     <IncludeAssemblyInVSIXContainer>false</IncludeAssemblyInVSIXContainer>
     <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>

--- a/Product/AllSDK/ItemTemplates/VisualBasic/Class/VBClassTemplateProject.csproj
+++ b/Product/AllSDK/ItemTemplates/VisualBasic/Class/VBClassTemplateProject.csproj
@@ -25,7 +25,7 @@
     <Configuration Condition="'$(FLAVOR_WIN)'!=''">$(FLAVOR_WIN)</Configuration>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <NetMFBuildVsSdkVersion>12</NetMFBuildVsSdkVersion>
+    <NetMFBuildVsSdkVersion>14</NetMFBuildVsSdkVersion>
     <GeneratePkgDefFile>false</GeneratePkgDefFile>
     <IncludeAssemblyInVSIXContainer>false</IncludeAssemblyInVSIXContainer>
     <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>

--- a/Product/AllSDK/ProjectTemplates/CSharp/EmulatorTemplate/EmulatorTemplateProject.csproj
+++ b/Product/AllSDK/ProjectTemplates/CSharp/EmulatorTemplate/EmulatorTemplateProject.csproj
@@ -21,7 +21,7 @@
     <Configuration Condition="'$(FLAVOR_WIN)'!=''">$(FLAVOR_WIN)</Configuration>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <NetMFBuildVsSdkVersion>12</NetMFBuildVsSdkVersion>
+    <NetMFBuildVsSdkVersion>14</NetMFBuildVsSdkVersion>
     <GeneratePkgDefFile>false</GeneratePkgDefFile>
     <IncludeAssemblyInVSIXContainer>false</IncludeAssemblyInVSIXContainer>
     <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>

--- a/Product/AllSDK/ProjectTemplates/CSharp/ExeTemplate/ExeTemplateProject.csproj
+++ b/Product/AllSDK/ProjectTemplates/CSharp/ExeTemplate/ExeTemplateProject.csproj
@@ -25,7 +25,7 @@
     <Configuration Condition="'$(FLAVOR_WIN)'!=''">$(FLAVOR_WIN)</Configuration>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <NetMFBuildVsSdkVersion>12</NetMFBuildVsSdkVersion>
+    <NetMFBuildVsSdkVersion>14</NetMFBuildVsSdkVersion>
     <GeneratePkgDefFile>false</GeneratePkgDefFile>
     <IncludeAssemblyInVSIXContainer>false</IncludeAssemblyInVSIXContainer>
     <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>

--- a/Product/AllSDK/ProjectTemplates/CSharp/LibTemplate/LibTemplateProject.csproj
+++ b/Product/AllSDK/ProjectTemplates/CSharp/LibTemplate/LibTemplateProject.csproj
@@ -25,7 +25,7 @@
     <Configuration Condition="'$(FLAVOR_WIN)'!=''">$(FLAVOR_WIN)</Configuration>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <NetMFBuildVsSdkVersion>12</NetMFBuildVsSdkVersion>
+    <NetMFBuildVsSdkVersion>14</NetMFBuildVsSdkVersion>
     <GeneratePkgDefFile>false</GeneratePkgDefFile>
     <IncludeAssemblyInVSIXContainer>false</IncludeAssemblyInVSIXContainer>
     <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>

--- a/Product/AllSDK/ProjectTemplates/CSharp/WinTemplate/WinTemplateProject.csproj
+++ b/Product/AllSDK/ProjectTemplates/CSharp/WinTemplate/WinTemplateProject.csproj
@@ -25,7 +25,7 @@
     <Configuration Condition="'$(FLAVOR_WIN)'!=''">$(FLAVOR_WIN)</Configuration>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <NetMFBuildVsSdkVersion>12</NetMFBuildVsSdkVersion>
+    <NetMFBuildVsSdkVersion>14</NetMFBuildVsSdkVersion>
     <GeneratePkgDefFile>false</GeneratePkgDefFile>
     <IncludeAssemblyInVSIXContainer>false</IncludeAssemblyInVSIXContainer>
     <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>

--- a/Product/AllSDK/ProjectTemplates/VB/EmulatorTemplate/VBEmulatorTemplateProject.csproj
+++ b/Product/AllSDK/ProjectTemplates/VB/EmulatorTemplate/VBEmulatorTemplateProject.csproj
@@ -21,7 +21,7 @@
     <Configuration Condition="'$(FLAVOR_WIN)'!=''">$(FLAVOR_WIN)</Configuration>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <NetMFBuildVsSdkVersion>12</NetMFBuildVsSdkVersion>
+    <NetMFBuildVsSdkVersion>14</NetMFBuildVsSdkVersion>
     <GeneratePkgDefFile>false</GeneratePkgDefFile>
     <IncludeAssemblyInVSIXContainer>false</IncludeAssemblyInVSIXContainer>
     <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>

--- a/Product/AllSDK/ProjectTemplates/VB/ExeTemplate/VBExeTemplateProject.csproj
+++ b/Product/AllSDK/ProjectTemplates/VB/ExeTemplate/VBExeTemplateProject.csproj
@@ -25,7 +25,7 @@
     <Configuration Condition="'$(FLAVOR_WIN)'!=''">$(FLAVOR_WIN)</Configuration>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <NetMFBuildVsSdkVersion>12</NetMFBuildVsSdkVersion>
+    <NetMFBuildVsSdkVersion>14</NetMFBuildVsSdkVersion>
     <GeneratePkgDefFile>false</GeneratePkgDefFile>
     <IncludeAssemblyInVSIXContainer>false</IncludeAssemblyInVSIXContainer>
     <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>

--- a/Product/AllSDK/ProjectTemplates/VB/LibTemplate/VBLibTemplateProject.csproj
+++ b/Product/AllSDK/ProjectTemplates/VB/LibTemplate/VBLibTemplateProject.csproj
@@ -25,7 +25,7 @@
     <Configuration Condition="'$(FLAVOR_WIN)'!=''">$(FLAVOR_WIN)</Configuration>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <NetMFBuildVsSdkVersion>12</NetMFBuildVsSdkVersion>
+    <NetMFBuildVsSdkVersion>14</NetMFBuildVsSdkVersion>
     <GeneratePkgDefFile>false</GeneratePkgDefFile>
     <IncludeAssemblyInVSIXContainer>false</IncludeAssemblyInVSIXContainer>
     <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>

--- a/Product/AllSDK/ProjectTemplates/VB/WinTemplate/VBWinTemplateProject.csproj
+++ b/Product/AllSDK/ProjectTemplates/VB/WinTemplate/VBWinTemplateProject.csproj
@@ -25,7 +25,7 @@
     <Configuration Condition="'$(FLAVOR_WIN)'!=''">$(FLAVOR_WIN)</Configuration>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <NetMFBuildVsSdkVersion>12</NetMFBuildVsSdkVersion>
+    <NetMFBuildVsSdkVersion>14</NetMFBuildVsSdkVersion>
     <GeneratePkgDefFile>false</GeneratePkgDefFile>
     <IncludeAssemblyInVSIXContainer>false</IncludeAssemblyInVSIXContainer>
     <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>

